server.js

```diff
@@ -59,6 +62,7 @@ app.use((req, res, next) => {
 // List All Routes here
 app.use("/", index);
 app.use("/users", users);
+app.use("/associations",associations);
```

Comment:
Comment out or eliminate this file, Its giving errors. You can't access this because you never made any paths.

Reply:
I commented that out just in case.